lib/shared/options/parser.js

```diff
@@ -63,60 +68,66 @@ module.exports = {
     type: 'number',
     requiresArg: true,
     default: undefined,  // To detect if this cli option is specified.
-    desc: ansi.gray(
+    desc: chalk.gray(
       'Specify the depth of the task dependency tree.'),
   },
   'compact-tasks': {
     type: 'boolean',
     default: undefined,  // To detect if this cli option is specified.
-    desc: ansi.gray(
+    desc: chalk.gray(
       'Reduce the output of task dependency tree by printing ' +
       'only top tasks and their child tasks.'),
   },
   'sort-tasks': {
     type: 'boolean',
     default: undefined,  // To detect if this cli option is specified.
-    desc: ansi.gray(
+    desc: chalk.gray(
       'Will sort top tasks of task dependency tree.'),
   },
   color: {
     type: 'boolean',
-    desc: ansi.gray(
+    desc: chalk.gray(
       'Will force gulp and gulp plugins to display colors, ' +
       'even when no color support is detected.'),
   },
   'no-color': {
     type: 'boolean',
-    desc: ansi.gray(
+    desc: chalk.gray(
       'Will force gulp and gulp plugins to not display colors, ' +
       'even when color support is detected.'),
   },
   silent: {
     alias: 'S',
     type: 'boolean',
     default: undefined,  // To detect if this cli option is specified.
-    desc: ansi.gray(
+    desc: chalk.gray(
       'Suppress all gulp logging.'),
   },
   continue: {
     type: 'boolean',
     default: undefined,  // To detect if this cli option is specified.
-    desc: ansi.gray(
+    desc: chalk.gray(
       'Continue execution of tasks upon failure.'),
   },
   series: {
     type: 'boolean',
     default: undefined,  // To detect if this cli option is specified.
-    desc: ansi.gray(
+    desc: chalk.gray(
       'Run tasks given on the CLI in series (the default is parallel).'),
   },
   'log-level': {
     alias: 'L',
     // Type isn't needed because count acts as a boolean
     count: true,
     default: undefined,  // To detect if this cli option is specified.
-    desc: ansi.gray(
+    desc: chalk.gray(
       'Set the loglevel. -L for least verbose and -LLLL for most verbose. ' +
       '-LLL is default.'),
-  },
+  }
 };
+
+var parser = yargs
+    .help(false).version(false).detectLocale(false)
```

Comment:
`.help(false)` and `.version(false)` is to prevent `yargs` from printing help and version automatically.
`.deleteLocale(false)` is to prevent `yargs` from translating some texts in help (e.g. type names) automatically according to the Locale.